pkg/interactor/data.go

```diff
@@ -0,0 +1,110 @@
+package interactor
+
+import (
+	"log"
+	"strconv"
+	"time"
+
+	. "github.com/Igoraamc/knot-cloud-storage/pkg/entities"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type DataDAO struct {
+	Server   string
+	Database string
+}
+
+var db *mgo.Database
```

Comment:
I might be wrong, but I believe we should be doing something like [this](https://github.com/bxcodec/go-clean-arch/blob/master/article/repository/mysql_article.go#L20), here :eyes: